src/callbacks/index.md

```diff
@@ -198,15 +199,15 @@ impl Accumulator {
 > `*mut Accumulator`, and then turn the mutable raw pointer into a mutable
 > reference.
 
-Now our `Accumulator` struct is defined and we have a `callback()` method which
-satisfies the `StatefulCallback` function signature, we can finally get on to
-using it.
+Now our `Accumulator` struct is defined and has a `callback()` method which
+satisfies the `StatefulCallback` function signature, now, it can finally
```

Comment:
How about rewording this to something like:

> Now that our `Accumulator` struct is defined and has a `callback()` method
> satisfying the `StatefulCallback` function signature, it is ready to be called
> by C.

I'm just trying to think of a way to avoid having the word "now" mentioned twice in the same sentence. Let me know if you can think of something better.